<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientClass</class>
 <widget class="QWidget" name="ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserRecv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>ClientClass</receiver>
   <slot>on_Connect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSend</sender>
   <signal>clicked()</signal>
   <receiver>ClientClass</receiver>
   <slot>on_Send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Connect_clicked()</slot>
  <slot>on_Send_clicked()</slot>
 </slots>
</ui>
